cmd/blockchaincmd/validators.go

```diff
@@ -1,124 +0,0 @@
-// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-package blockchaincmd
-
-import (
-	"errors"
-	"os"
-	"strconv"
-	"time"
-
-	"github.com/ava-labs/avalanche-cli/pkg/cobrautils"
-	"github.com/ava-labs/avalanche-cli/pkg/models"
-	"github.com/ava-labs/avalanche-cli/pkg/networkoptions"
-	"github.com/ava-labs/avalanche-cli/pkg/subnet"
-	"github.com/ava-labs/avalanchego/ids"
-	"github.com/ava-labs/avalanchego/vms/platformvm"
-	"github.com/olekukonko/tablewriter"
-	"github.com/spf13/cobra"
-)
-
-// avalanche blockchain validators
-func newValidatorsCmd() *cobra.Command {
```

Comment:
we already have avalanche validators list

Reply:
we need to integrate this into validators list, then remove. this is for another PR